packages/react-hooks/src/useIpfsFetch.ts

```diff
@@ -68,7 +78,7 @@ export function useIpfsFetch <T> (hashes: string[] | undefined, options?: Option
     }
 
     return hashes
-      .map((hash) => isCid(hash) ? hash : '')
+      .map((hash) => isCid(hash) ? extractCid(hash) : '')
```

Comment:
Sorry, bit out of it and took me ages to get here:

Doesn't this mean that we lose support for the old format? Or is the new format the only way things are done now?

(Just need to do a search for the usage of this hook as well - it _may_ affect other places, e.g. I think it is used in the files app?)

Reply:
I'll double-check the code if affects other places (as I remember, it didn't, but want to be sure).
Also, I think we can write a better regexp here, will check next week.

In general, we won't lose the support of the old format, on the contrary, we failed to extract the cid if it was stored in a format with prefixes `ipfs://` or `ipfs://ipfs/`, this PR fixes that.

Reply:
Perfect.

Reply:
Done. I've reworked the whole metadata fetching approach, and now we support both ipfs cid versions + metadata stored on regular http* servers